app/javascript/packages/document-capture/components/documents-step.jsx

```diff
@@ -84,6 +85,10 @@ function DocumentsStep({
                 [`${side}_image_metadata`]: JSON.stringify(metadata),
               })
             }
+            onCameraAccessDeclined={() => {
```

Comment:
Need this on `ReviewIssuesStep` as well.

Reply:
Maybe it's time to extract a new component here, like `<DocumentSideAcuantCapture />`, since it seems to be common to overlook that these are mostly in-sync across "Documents" and "Review Issues" steps.

Reply:
> Maybe it's time to extract a new component here, like `<DocumentSideAcuantCapture />`, since it seems to be common to overlook that these are mostly in-sync across "Documents" and "Review Issues" steps.

Done in 6fc2e7b.